.travis.yml

```diff
@@ -0,0 +1,41 @@
+# Config file for automatic testing at travis-ci.org
+
+language: python
+
+python:
+  - 2.7
+  - 3.5
+
+env:
+  - TOXENV=django18
+  - TOXENV=django19
+  - TOXENV=django110
+
+matrix:
+  include:
+    - python: 3.5
+      env: TOXENV=quality
+    - python: 3.5
+      env: TOXENV=docs
+
+cache:
+  - pip
+
+install:
+  - pip install -r requirements/travis.txt
+
+script:
+  - tox
+
+after_success:
+  - codecov
+
+# Set password via "travis encrypt --add deploy.password"; for details, see
+# https://docs.travis-ci.com/user/deployment/pypi
+deploy:
+  provider: pypi
+  user: edx
+  distributions: sdist bdist_wheel
+  on:
+    tags: true
+    condition: '$TOXENV = py35-django19'
```

Comment:
This was a bug in the cookiecutter that I'm fixing now - this TOXENV isn't used in travis runs, should be changed to `quality`.